slider/library.py

```diff
@@ -53,7 +53,7 @@ def keys(self):
 if sys.platform.startswith('win'):
     def sanitize_filename(name):
         for invalid_character in r':*?"\/|<>[]':
-            name.replace(invalid_character, '')
```

Comment:
That will do it. One other thing is that we are stripping too many characters, should we remove the ones that are not actually invalid?